jack.py

```diff
@@ -1247,6 +1248,98 @@ def callback_wrapper(_):
+        the timebase may be done conditionally, so it fails if there was
+        a master already.
+
+        The method may be called whether the client has been activated
+        or not.
+
+        Parameters
+        ----------
+        callback : callable
+            Realtime function that returns extended position
+            information. Its output affects all of the following process
+            cycle. This realtime function must not wait.
+
+            This function is called immediately after `process()` in the
+            same thread whenever the transport is rolling, or when any
+            client has requested a new position in the previous
+            cycle. The first cycle after `set_timebase_callback()` is
+            also treated as a new position, or the first cycle after
+            `jack_activate()` if the client had been inactive.
+
+            The timebase master may not use its `pos` argument to set
+            `pos.frame`. To change position, use
+            `jack_transport_reposition()` or
+            `jack_transport_locate()`. These functions are
+            realtime-safe, the `timebase_callback` can call them
+            directly.
+
+            It must have this signature::
+
+                callback(state:int,
+                         frames:int,
+                         pos:CFFI_struct,
+                         new_pos:bool) -> None
+
+            * state : int
+                current transport state.
+            * frames : int
+                number of frames in current period.
+            * pos : Position CFFI structure
+                position structure for the next cycle; `pos.frame` will
+                be its frame number. If `new_pos` is `False`, this
+                structure contains extended position information from
+                the current cycle. If `True`, it contains whatever was
+                set by the requester. The `timebase_callback`'s task is
+                to update the extended information here. See `JACK
+                transport documentation`__ for the available fields.
+            * new_pos : bool
+                `True` (non-zero) for a newly requested `pos`, or for
+                the first cycle after the `timebase_callback` is
+                defined.
+        conditional : bool
+            set to `True` for a conditional request. If set to `True`
+            this call will fail if there is already a timebase master
+
+        Returns
+        -------
+        bool
+            `True` if new timebase master was set. `False` if
+            conditional request fail bacause another master is already
+            registered.
+
+__ http://jackaudio.org/files/docs/html/structjack__position__t.html
+
+        """
+        @self._callback("JackTimebaseCallback")
+        def callback_wrapper(state, frames, pos, new_pos, _):
+            return callback(state, frames, pos, new_pos)
+
+        return_code = _lib.jack_set_timebase_callback(self._ptr,
```

Comment:
I just realized that if you change the name according to my first comment, my second comment becomes obsolete, so just forget it!